addons/packages/pinniped/0.12.1/bundle/config/values.star

```diff
@@ -10,7 +10,13 @@ def validate_pinniped():
   data.values.tkg_cluster_role in ("management", "workload") or assert.fail("tkg_cluster_role must be provided to be either 'management' or 'workload'")
   data.values.infrastructure_provider in ("vsphere", "azure", "aws") or assert.fail("infrastructure_provider must be provided to be either 'vsphere', 'azure' or 'aws'")
   if data.values.identity_management_type:
-    data.values.identity_management_type in ("oidc", "ldap") or assert.fail("identity_management_type must be provided to be either 'oidc' or 'ldap'")
+
```

Comment:
Our user story is to support `identity_management_type: none` only on the workload cluster. This is the point of enforcement and forking behavior for mgmt cluster.